models/staging/stg_ga4_export__event.sql

```diff
@@ -70,17 +70,9 @@ final_pre as (
 
     from fields
 
-),
-
-exclude_intraday as (
-
-    select
-        *
-    from final_pre
-    where is_intraday = false
-
 )
 
 select
     *
-from exclude_intraday
+from final
+where not is_intraday
```

Comment:
Typically we will do a coalesce here to default the values when `null`. However, I noticed in our own data that the valid fields are in fact `null` and not `true`. Should we apply any coalesce here or just leave it as is since this is how the data is synced via the connector.

Reply:
Also, just consolidated this into the final cte instead of creating a separate one as it felt a bit duplicative.